### PR TITLE
Add pin code and workflow id to verify api and test

### PR DESCRIPTION
**Pull request by lshep311**
- Add pin code and workflow id to StartVerificationRequest
- Add verify_test

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -10,6 +10,7 @@ github.com/jarcoal/httpmock v1.0.4 h1:jp+dy/+nonJE4g4xbVtl9QdrUNbn6/3hDT5R4nDIZn
 github.com/jarcoal/httpmock v1.0.4/go.mod h1:ATjnClrvW/3tijVmpL/va5Z3aAyGvqU3gCT8nX0Txik=
 github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZbAQM=
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
+github.com/stretchr/objx v0.1.0 h1:4G4v2dO3VZwixGIRoQ5Lfboy6nUhCyYzaqnIAPPhYs4=
```
Comment (remitly-srivatsa) [inline go.sum:13]: why was this change needed?

Comment (remitly-srivatsa) [reply go.sum]: ah, never mind, it looks like it was pulled in from the assert

Comment (lshep311) [reply go.sum:13]: I didn't explicitly change it - just happened when I pulled it down and my golang evaluated dependencies

File: verify_test.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+	_, _, err := _client.Verify.Start(StartVerificationRequest{
+		Number:      "447520615146",
+		Brand:       "NEXMOTEST",
+		PINCode:     "1234",
+		RequireType: "all",
+		WorkflowID:  4,
+	})
+
+	assert.Nil(t, err)
```
Comment (remitly-srivatsa) [inline verify_test.go:29]: should we also add a 

```
	assert.Equal(t, "0", response.Status)
```

**Review by remitly-srivatsa — APPROVED** [review]
LGTM

File: verify_test.go (hunk 2 above)
Comment (lshep311) [reply verify_test.go:29]: Sure, I have added that
